### ISSUE TITLE
Improve accessibility of the feedback form

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons#14741]
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1864523, we need to update the feedback form to improve its accessibility. There are things that we cannot address, though, mainly around the radio buttons and how we group them ("1.  Keyboard navigation:" and "2. Grouping:"). That's because we want a single group of radio buttons.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDFRNT-295)
